### DISCUSSION TITLE
What's the correct way to use `intersection`?

### DISCUSSION THREAD

**Discussion by GalinhaLX** [mavoweb/mavo#921] [Q&A]
Hi

In this [codepen](https://codepen.io/galinhalx/pen/PoeGKVG),
I'm trying to check if a list intersects with another list, and retrieve which properties/values intersect

I'm able to use `intersects`, but not `intersection`

What's the correct way to use `intersection`? 

By the way, my goal is:
A restaurant have special menu offers, composed of different meals

- Starter + main course
- main course + desert
- Starter + main course + desert

=> in case a user orders meals, which combination (eg: main course + desert) matchs with a menu, 

I would like to 

1. display items of intersection
2. retreive the menu's corresponding price
3. display the items that are out of intersection, in case user adds extra items not in menu (can't find a mavo function to achieve this)

All ideas to achieve this are welcome

**Comment by DmitrySharabin** [comment]
I'm afraid we have a bug here. Thank you for finding it!

I filed an issue and submitted a fix. Unfortunately, it might take some time to push it since Lea is extremely busy right now, and I don't want to bother her.

If it's crucial for you, I would suggest testing your use case with this version of Mavo till we fix the issue: https://deploy-preview-923--getmavo.netlify.app/mavo.js
